generate-db.js

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env node
 const fs = require("node:fs");
-const https = require("node:https");
```

Comment:
Why are you moving from https to http2? http2 does not require encryption https://http2.github.io/faq/#does-http2-require-encryption which can be a problem

Reply:
Oh I didn't go deeper with that. I think the URL is still https so it is still encrypted. Did I get anything wrong?